ddtrace/_trace/span.py

```diff
@@ -508,6 +509,15 @@ def _get_baggage_item(self, key: str) -> Optional[Any]:
         """Gets a baggage item from the span context of this span."""
         return self.context._get_baggage_item(key)
 
+    def _get_all_baggage_items(self) -> str:
```

Comment:
Can we remove all the baggage getters and setters from the span object. We can use `span.context._get/set_baggage` everywhere